gulpfile.ts

```diff
@@ -9,7 +9,7 @@ loadTasks(PROJECT_TASKS_DIR);
 
 // --------------
 // Build dev.
-gulp.task('build.dev', done =>
+gulp.task('build.dev', (done: any) =>
```

Comment:
I'm not a big fan of this `any`. Wouldn't it be better to add an `interface.ts` somewhere and have `ICallback`
```ts
export interface ICallback {
  (): void;
}
```